CWE/events/Essential Events 4.txt

```diff
@@ -2283,22 +2283,88 @@ country_event = {
 	picture = "franco_rally"
 
 	trigger = {
-OR = { tag = SPA tag = POR }
-NOT = { year = 1990 }
-government = populist_dictatorship
-ruling_party_ideology = nationalist
-NOT = { has_country_flag = franco_boost }
+        OR = { tag = SPA tag = POR }
+        NOT = { year = 1990 }
+        government = populist_dictatorship
+        ruling_party_ideology = nationalist
+        NOT = { has_country_flag = franco_boost }
 	}
 
 	option = {
 		name = "Their support will not be let down!"
-set_country_flag = franco_boost
-any_pop = { ideology = {
-	factor = 0.7
-	value = nationalist
-} }
-					}
+		set_country_flag = franco_boost
+		any_pop = {
+			limit = { AND = { is_accepted_culture = yes strata = rich } }
```

Comment:
is_accepted_culture = yes doesn't include is_primary_culture = yes so OR = { is_primary_culture = yes is_accepted_culture = yes } would be better